examples/multi-account-multi-region/provider.tf

```diff
@@ -11,3 +11,28 @@ provider "aws" {
   access_key = var.aws_peer_access_key
   secret_key = var.aws_peer_secret_key
 }
+
+# # if you have ~/.aws/credentials as below
+# [this_account]
+# region                = ap-southeast-2
+# aws_secret_access_key = xxxx
+# aws_access_key_id     = xxxx
+#
+# [peer_account]
+# region                = ap-southeast-2
+# aws_secret_access_key = xxxx
+# aws_access_key_id     = xxxx
+#
+# you can adjust to
+#
+# provider "aws" {
+#   alias      = "this"
+#   region     = "us-east-1"
+#   profile    = "this_account"
+# }
+#
+# provider "aws" {
+#   alias      = "peer"
+#   region     = "us-west-1"
+#   profile    = "peer_account"
+# }
```

Comment:
No idea for what propose you add this section

Reply:
because i don’t like that the solution to ask other to save secrets in source codes.

Use aws profile is the best practice